packages/UI/src/components/molecules/Alert/Alert.tsx

```diff
@@ -21,7 +22,9 @@ const Alert: React.FC<React.PropsWithChildren<AlertProps>> = ({
         <></>
       )}
       <Typography css={{ flex: 1 }}>{children}</Typography>
-      <Icons.Close />
+      <button>
+        <Icons.Close onClick={handleClose} />
```

Comment:
@helioLJ this is a great addition! thanks for adding handleClose prop.
can you please make the onClick on the button rather than the Icon

Reply:
Sure! I it will be more concise this way, ty